### PR TITLE
added space

### PR DESCRIPTION
**Pull request by AMarcedone**

### CODE REVIEW

File: shared/provision/code-page/index.tsx [hunk 1]
```diff
@@ -379,7 +379,7 @@ const Instructions = (p: Props) => (
     {p.currentDeviceAlreadyProvisioned ? (
       <Kb.Box2 direction="horizontal" centerChildren={true}>
         <Kb.Text center={true} type={textType} style={styles.instructions}>
-          Ready to authorize using{' '}
+          Ready to authorize using&nbsp;
```
Comment (jakob223) [inline shared/provision/code-page/index.tsx]: does this work on mobile?

**Comment by jakob223** [comment]
CI failure is because you need to update storyshots. `yarn test -u stories`

**Comment by jzila** [comment]
I don't think this will work on mobile.

**Comment by AMarcedone** [comment]
Admittedly I haven't tested it, but @thebearjew claimed it would.

**Comment by thebearjew** [comment]
@amarcedone you should be testing on mobile anyway.

**Comment by AMarcedone** [comment]
Ok, I'll try to check, but getting an android dev environment running will
take some time.

**Comment by jzila** [comment]
Just tested, actually seems ok.

**Pull request merged**